docs/fsharp/language-reference/query-expressions.md

```diff
@@ -2432,4 +2432,4 @@ end
 
 - [F# Language Reference](index.md)
 - [QueryBuilder Class](https://fsharp.github.io/fsharp-core-docs/reference/fsharp-linq-querybuilder.html)
-- [Computation Expressions](Computation-Expressions.md)
+- [Computation Expressions](computation-expressions.md)
```

Comment:
While things like this doesn't affect the correctness of link in live site, it should affect someone working locally on Linux since paths are case-sensitive.